docs/identities/sign-in/actions.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: actions
+title: Customize login and registration flows with webhooks and actions
+sidebar_label: Actions and webhooks
```

Comment:
So... `Actions` aren't our name for webhooks/hooks? I'm confused.

Reply:
Webhooks are a type of action :) An action is also: invalidate session after login, require verification, ...

Developers usually search for webhook if they want to make an HTTP RPC call, therefore I think it's worth mentioning it here.

/cc @tomekpapiernik

Reply:
I understand the intention, but the execution makes it confusing I think. Even later in this doc you mention `Ory Actions` as if it were its own thing, the sidebar and title say "Actions **and** webhooks" which makes it seem that webhooks aren't Actions...